include/klee/Internal/ADT/TreeStream.h

```diff
@@ -18,6 +18,19 @@ namespace klee {
   typedef unsigned TreeStreamID;
   class TreeOStream;
 
+  struct PathLocation {
```

Comment:
Right now, we discard everything from the .path file during replay except for the branch decision.   So we're paying the price for extra memory consumption, without getting any benefits.  As I mentioned before, we should indeed read the location back and use it to double-check that the replay proceeds correctly.  However, we could leave this for later and just change `PathLocation` to have a reference to a `KInstruction`.  @jirislaby , would you be happy with this decision?